Networking/NetworkingTests/Responses/systemStatus.json

```diff
@@ -329,8 +329,26 @@
             }
         ],
         "dropins_mu_plugins": {
-            "dropins": [],
-            "mu_plugins": []
+            "dropins": [
+                {
+                    "plugin": "advanced-cache.php",
+                    "name": "advanced-cache.php"
+                },
+                {
+                    "plugin": "object-cache.php",
+                    "name": "Memcached"
+                }
+            ],
+            "mu_plugins": [
+                {
+                    "plugin": "wpcomsh-loader.php",
+                    "name": "WP.com Site Helper",
+                    "version": "",
+                    "url": "",
+                    "author_name": "",
+                    "author_url": ""
```

Comment:
nit: it looks like `mu_plugins` have more info than just plugin and name as in `dropins`, does it make sense to decode `mu_plugins` with `SystemPlugin` to include all the info? We can make the missing fields optional?

Reply:
Since the other fields are empty and from what I gathered from Core's reports, only the plugin and the name are displayed so I'll keep the simple model in this case 😄